.travis.yml

```diff
@@ -1,49 +1,22 @@
 language: python
 python:
-# This is not actually used. Because it would take an overly long time
-# to build scipy we cannot use the virtual env of travis. Instead, we
-# use miniconda.
   - "2.7"
+  - "3.4"
+  - "3.5"
+  - "3.6"
+  - "3.7-dev"
```

Comment:
It always worked without miniconda (see circle config and install instructions) but since @QuLogic went through the effort of setting it up with conda, we retained it in travis. I don't know enough about how to change the conda setup to test the various versions, so this seemed the simplest solution.

Reply:
The conda environment is created with `python=$TRAVIS_PYTHON_VERSION`, which comes from this key; you didn't really need to re-write everything to get it to work.

But originally, miniconda was only necessary because SciPy took forever to compile from source; I assume there are wheels now.

Also, 3.7 is GA, you should use it and not 3.7-dev which is a very snapshot.